src/classes/RD2_EnablementService_TEST.cls

```diff
@@ -51,14 +51,16 @@ public with sharing class RD2_EnablementService_TEST {
     * when enhanced Recurring Donations are not enabled
     */
     @isTest
-    private static void shouldCreateNewOppOnInsertWhenEnhancementIsNotEnabled() {
+    private static void shouldCreateNewOppOnInsertWhenEnhancementIsNotEnabledWhenRD2TriggerIsDisabled() {
```

Comment:
Is it valid to rename to `shouldCreateNewOppOnInsertWhenRD2TriggerIsDisabled()`?

Reply:
I do agree that the method name should be changed back. 

As per our quip chat, the RD2_RecurringDonation_TDTM trigger will always be executed via TDTM - even if the `RecurringDonationSettings__c.IsRecurringDonations2Enabled__c` field is set to false. This method can validate the code isolation by making sure that there are no RecurringDonatinoSchedule records created by the RD2 trigger if RD2 is not enabled.